src/App.svelte

```diff
@@ -1,82 +1,82 @@
 <script lang="ts">
-	import Counter from "./counter.svelte";
-	import { counterItems } from "./store"
+  import Counter from "./counter.svelte";
 
-	interface CounterItems {
-		countername: string;
-		count: number;
-	}
+  interface CounterItems {
+    countername: string;
+    count: number;
+  }
 
-	function addCounter() {
-    	$counterItems = [...$counterItems, {countername: "new", count: 0}]
-  	}
-	
-	function sumCount(Items: CounterItems[]) {
-		let sumcount: number = 0;
-		Items.forEach(element => {
-			sumcount += element.count;
-		});
-		return sumcount;
-	}
+  let counterItems: CounterItems[] = [{ countername: "new", count: 0 }];
 
-	$: sum = sumCount($counterItems);
+  function addCounter() {
+    counterItems = [...counterItems, { countername: "new", count: 0 }];
+  }
+
+  function sumCount(Items: CounterItems[]) {
+    let sumcount = 0;
+    Items.forEach((element) => {
+      sumcount += element.count;
+    });
+    return sumcount;
+  }
+
+  $: titlelist = counterItems.map((Item) => {
+    return Item.countername;
+  });
```

Comment:
{}をなくせば、returnなくせるので、行数減らせそうですね 🙆‍♂️ 

```suggestion
  $: titlelist = counterItems.map((Item) => Item.countername)
```